week-3-practise/exercise-1.js

```diff
@@ -0,0 +1,11 @@
+/* The following numbers are ages for a group of students. Create a function so that you’ll find out their ages “in total” after 10 years
+Hint: Use find method */
+var numbers = [18, 17, 19, 16];
+var checkingAges = numbers.map(number => number + 10);
+var totalAgeInTotal = checkingAges.reduce((a, b) => a + b, 0);
```

Comment:
Wow! Great usage of the `.reduce()` method! This is a very powerful but quite complex method